tagmedia

```diff
@@ -1,25 +1,43 @@
 #! /usr/bin/perl
 
-# Add/remove key=value pairs from iso application area (0x200 bytes at
-# 0x8373). Entries are separated by semicolons ';'.
+# Add/remove key=value pairs from iso application area (0x200 bytes starting
+# at 0x8373). Entries are separated by semicolons (';').
```

Comment:
is possible to add here link about this iso format definition ( so easy to find why 0x200 at 0x8373 ).

Reply:
I thought I had added it somewhere...